engine/src/license.h

```diff
@@ -126,4 +126,38 @@ inline bool MCStringFromLicenseClass(MCLicenseClass p_class, bool p_simplified,
     return false;
 }
 
+static const struct { const char *tag; MCLicenseClass value; } s_edition_map[] =
+{
+    { "community", kMCLicenseClassCommunity },
```

Comment:
@livecodeian my only comment on this is to wonder if we should just have a single struct for both APIs to iterate by just adding `edition_tag` or something to the other one. Just to keep the map between one string value and the other clear in the code.

Reply:
@livecodeian just poking you to see what your thoughts are on the above. If you don't think it's worthwhile let me know and I'll review this ok